code/game/objects/items/wayfinding.dm

```diff
@@ -23,40 +29,34 @@
 
 	synth_acc.account_holder = name
 
-	desc += " Only [ppt_cost] credits! It also likes making costumes..."
+	desc += " [ppt_cost ? "Only [ppt_cost] credits! " : ""]It also synthesises costumes for some reason."
 
-	set_expression("neutral")
+	set_expression("happy") //actual first law of robotics it needs to be friendly in an insincere way
 
 /obj/machinery/pinpointer_dispenser/attack_hand(mob/living/user)
 	if(world.time < user_interact_cooldowns[user.real_name])
-		to_chat(user, "<span class='warning'>It doesn't respond.</span>")
+		set_expression("veryhappy", 2 SECONDS)
```

Comment:
Could have useful feedback. Would be good if you mentioned that a cooldown existed.

Reply:
Okay, that's probably right.